constants.py

```diff
@@ -98,6 +98,13 @@ def error_string(error, logname, with_hd):
 logsfs_label_prefix = 'logs-'
 logsfs_label = logsfs_label_prefix + '%s'
 
+rootpart_label = "root"
+backuppart_label = "backup"
+lvmpart_label = "Local SR"
```

Comment:
I think this should be `storagepart_label` since it doesn't necessarily use LVM.